ci/Dockerfile

```diff
@@ -5,6 +5,9 @@ ARG GINKGO_VERSION=latest
 ARG GOLANGCILINT_VERSION=latest
 RUN apt-get update && apt-get install -y libpcap-dev && rm -rf /var/lib/apt/lists/*
 
+RUN curl -fsSL https://deb.nodesource.com/setup_current.x | sudo -E bash - && \
+    apt-get install -y nodejs && rm -rf /var/lib/apt/lists/*
```

Comment:
Consider installing semantic-release here right away, see also comment below in `shipit`, where it is installed every time.

Reply:
I did meanwhile manage to add everything to the dockerfile.